### PR TITLE
Make strings in midiEditorCommands.cpp translatable (#560)

### PR DESCRIPTION
**Pull request by RDMurray**
I think this covers all of the strings that should be translated in midiEditorCommands.cpp.

### CODE REVIEW

File: src/midiEditorCommands.cpp [hunk 1]
```diff
@@ -1443,17 +1547,18 @@ void postMidiSwitchCCLane(int command) {
 }
 
 void postToggleMidiInputsAsStepInput(int command) {
-	ostringstream s;
-	s << (GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION),
-		command) ? "enabled" : "disabled");
-	s << " MIDI inputs as step input";
-	outputMessage(s);
+	bool enabled = GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION), command);
+	if(enabled){
```
Comment (jcsteh) [inline src/midiEditorCommands.cpp:1551]: Out of curiosity, why the bool variable here but not later?

**Review by jcsteh — CHANGES_REQUESTED** [review]
Thank you very much.

**Comment by RDMurray** [comment]
I don't know how to get the individual reply fields above to work.  

with regard to cmdMidiNoteSplitOrJoin,
Currently those are the only two actions that can call this function.  Maybe in the future there could be others, but we could add them at the time as "transformed" might not make sense anyway.

File: src/midiEditorCommands.cpp [hunk 2]
```diff
@@ -960,7 +976,9 @@ void cmdMidiMoveToTrack(Command* command) {
 	char* trackName = (char*)GetSetMediaTrackInfo(track, "P_NAME", NULL);
 	if (trackName)
 		s << " " << trackName;
-	s << " item " << itemNum << " " << GetTakeName(take);
+	s << " " << format(
```
Comment (jcsteh) [inline src/midiEditorCommands.cpp:979]: This one needs a translators comment.
```suggestion
	// Translators: Used when reporting activation of the next/previous track in
	// the MIDI editor. {num} will be replaced with the item number. {name} will
	// be replaced with its name. For example: "item 2 chorus".
	s << " " << format(
```

**Review by jcsteh — APPROVED** [review]
Thanks! I particularly appreciate this given how tedious it is. :)

File: src/midiEditorCommands.cpp [hunk 3]
```diff
@@ -1443,17 +1553,18 @@ void postMidiSwitchCCLane(int command) {
 }
 
 void postToggleMidiInputsAsStepInput(int command) {
-	ostringstream s;
-	s << (GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION),
-		command) ? "enabled" : "disabled");
-	s << " MIDI inputs as step input";
-	outputMessage(s);
+	bool enabled = GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION), command);
+	if(enabled){
```
Comment (jcsteh) [inline src/midiEditorCommands.cpp:1557]: Your update seems to have gotten lost here somehow...
```suggestion
	if (GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION), command)) {
```

**Pull request merged**